Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputMapper.cpp

```diff
@@ -366,7 +408,16 @@ namespace AzToolsFramework
                     buttonChannel->UpdateState(false);
                 }
 
-                NotifyUpdateChannelIfNotIdle(buttonChannel, mouseEvent);
+                if (m_lastConsumedInputChannelIdCrc32 == buttonChannel->GetInputChannelId().GetNameCrc32())
```

Comment:
This  (and the other location)could use a comment - the logic makes sense to me now, but it's non-obvious that `m_lastConsumedInputChannelIdCrc32` will be updated here.

Reply:
That's fair, will update the comments a few lines above where the resetting happens to mention how it gets updated.